lv_conf_template.h

```diff
@@ -273,7 +273,7 @@
  *Used by image decoders such as `lv_lodepng` to keep the decoded image in the memory.
  *Data larger than the size of the cache also can be allocated but
  *will be dropped immediately after usage.*/
-#define LV_CACHE_DEF_SIZE       0
+#define LV_CACHE_DEF_SIZE       32
```

Comment:
Please add what the unit is. (byte, kB, etc)

Reply:
Currently it's based on count, it's working fine now, I'll add the cache method based on memory size at a later stage

Reply:
I think it's an important step, so please add the size based operation before merging it.